src/BlazorWebView/src/WindowsForms/Microsoft.AspNetCore.Components.WebView.WindowsForms.csproj

```diff
@@ -21,7 +21,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="build\**" CopyToOutputDirectory="Never" CopyToPublishDirectory="Never" Pack="true" PackagePath="build\" />
+    <Content Remove="build\**\*" />
+    <None Include="build\**\*" Pack="True" PackagePath="build\%(RecursiveDir)%(FileName)%(Extension)" />
```

Comment:
I copied what we're doing for the MAUI BlazorWebView package to here and it works.

Reply:
It might be that the stuff in the build folder might have only been Content in the msbuild world? So the None Update never actually applied. You can check the binlogs to confirm. But, if this is working then it doesn't matter. It is just the pack.

Did you check the packages on CI artifacts to make sure it is correct?

Reply:
Yup I compared packages on CI before & after this. Before this change (I looked at some random other build), the targets were missing. On the CI build for this PR the targets are there.